test/testcafe/spec/EnrollAuthenticatorPasswordView_spec.js

```diff
@@ -70,7 +77,19 @@ test('should succeed when fill same value', async t => {
     .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
 });
 
-test('should have the correct reqiurements', async t => {
+test.requestHooks(errorMock)('should show a callout when server-side field errors are received', async t => {
```

Comment:
add another test for client side validation ?

Reply:
I have added above - https://github.com/okta/okta-signin-widget/pull/2028/files#diff-d02cad9b7fcfc4a5bd52981fb90019e6a71df9265f3fb2b7303a35979c92d085R62